starknet_py/tests/e2e/client/conftest.py

```diff
@@ -19,18 +19,21 @@
     prepare_net_for_tests,
     PreparedNetworkData,
 )
-from starknet_py.tests.e2e.conftest import contracts_dir
+from starknet_py.tests.e2e.conftest import contracts_dir, AccountToBeDeployedDetails
 
 directory = os.path.dirname(__file__)
 
 
 async def prepare_network(
     new_gateway_account_client: AccountClient,
+    deploy_account_details: AccountToBeDeployedDetails,
```

Comment:
Is it really from this PR or just the wrong branch target?

Reply:
Branch target. Will rebase this once 0.10.1-docs is merged.